### DISCUSSION TITLE
API for Account health metrics - ODR, VTR, Feedback and Suppression

### DISCUSSION THREAD

**Discussion by AliMohammedSyed** [amzn/selling-partner-api-models#3539] [General]
Hi guys. I am trying to find the APIs in SP API which can give me the data necessary for 
API for Account health metrics - ODR, VTR, Feedback and Suppression. 

Also is there an API which can give us export of all the reviews including positive reviews on amazon?

**Comment by david-rgb** [comment]
Hi, did you manage to get any information regarding this issue. I am also interested in this.
